### DISCUSSION TITLE
i am trying to use ace but some issue facing the cursor stops and line are overlapping it is window system.

### DISCUSSION THREAD

**Discussion by Achal4004** [ajaxorg/ace#5030] [Q&A]
<img width="485" alt="image" src="https://user-images.githubusercontent.com/51490652/213534055-e0ff3cb3-dd4d-4918-8ddd-d59cfb9b4108.png">

**Comment by nightwing** [comment]
Are there any errors displayed in the browser console? And could you put your code on jsbin to create a demo reproducing the bug.

**Reply by Achal4004** [reply]
<img width="578" alt="image" src="https://user-images.githubusercontent.com/51490652/213643371-53bb4f5a-e4ef-4907-9c85-b9ec5604b3c8.png">

**Reply by nightwing** [reply]
This image doesn't show the browser devtools console which can be opened using F12 key, or from the context menu https://developer.chrome.com/docs/devtools/console/

**Comment by Achal4004** [comment]
actually it is happing when i am opening it in mat dialog modal in angular like this

<img width="578" alt="image" src="https://user-images.githubusercontent.com/51490652/213643234-a5f4d074-fb3a-4219-be9d-b0042fc76583.png">
